provider_shopper/lib/screens/cart.dart

```diff
@@ -65,7 +65,7 @@ class _CartTotal extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     var hugeStyle =
-        Theme.of(context).textTheme.headline1.copyWith(fontSize: 48);
+        Theme.of(context).textTheme.headline1!.copyWith(fontSize: 48);
```

Comment:
I think this is unsafe, although probably fine for the purposes of this particular sample. We don't want to spend too much time on minutiae when the star of the show should be state management.

Reply:
Let's keep this as is.

Reply:
Sure thing!